lib/js/components/SectionHeader/SectionHeader.vue

```diff
@@ -84,23 +84,29 @@ $icons-and-slot-min-height-l: 50px;
 		color: $color-neutral-text;
 	}
 
+	&__slot,
+	&__iconWrapper {
+		display: flex;
+		align-items: center;
+	}
+
+	&__iconWrapper {
+		min-height: $icons-and-slot-min-height-m;
```

Comment:
why don't you center icon using flex?
`align-self: center` should do the trick

Reply:
We can't, because of this case where text will have more than 1 line, it's UX recommendation:
![Screenshot 2022-12-01 at 15 19 56](https://user-images.githubusercontent.com/4208462/205076541-fc8b69c3-1422-48f5-8a01-94e6b397f65d.png)